app/models/embedded_ansible_worker/object_management.rb

```diff
@@ -21,16 +21,16 @@ def ensure_organization(provider, connection)
     return if provider.default_organization
 
     provider.default_organization = connection.api.organizations.create!(
-      :name        => I18n.t("product.name"),
-      :description => "#{I18n.t("product.name")} Default Organization"
+      :name        => Vmdb::Appliance.PRODUCT_NAME,
+      :description => "#{Vmdb::Appliance.PRODUCT_NAME} Default Organization"
```

Comment:
This looks like a missing i18n not getting any better with this PR

ping @mzazrivec

Reply:
I think this part of the backend code creates objects in the embedded ansible instance and I'd say there's no point in applying i18n here.

Reply:
@carbonin Can you just review this particular file just in case?